src/components/network/PageNetworks.vue

```diff
@@ -1,6 +1,7 @@
 <template>
   <TmPage data-title="Network" class="page" hide-header>
```

Comment:
I will accept the PR but want to show you that TmPage already has logic for loading states. Check PageValidator

Reply:
Yes, I saw! Thank you for the hint :wink: !